jira-mixin/alerts/general.yaml

```
@@ -0,0 +1,43 @@
+groups:
+- name: alert.rules
+  rules:
+  - alert: InstanceDown
+    expr: up == 0
+    for: 1m
+    labels:
+      severity: "critical"
+    annotations:
+      summary: "JIRA server down"
+      description: "The JIRA server has been down for more than 1 minute."
+  - alert: LicenseExpired
+    expr: jira_maintenance_expiry_days_gauge == 0

```

I wonder if this gauge might be negative if the license has expired in the past...

How about:

```suggestion
    expr: jira_maintenance_expiry_days_gauge <= 0
```

---

I'm not sure of the functionality for that, so that's a good point. I updated it to be <= 0